src/Benchmarks/Startup.cs

```diff
@@ -13,6 +14,7 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+            services.AddSingleton(typeof(IHttpContextAccessor), typeof(InertHttpContextAccessor));
```

Comment:
Yeah I definitely get it. Very excited to this what the numbers will look like!

My comments are more about finding a long-term alternative than about the specifics.